### PR TITLE
Deployer enabled

### PR DESCRIPTION
**Pull request by dhidalgX**
@gabmontes Could we approve this PR before deploying the latest commit? 
Theres some ENV variables i have added to they wont get overwritten and also enabling the deployer.

Thanks!

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -1,9 +1,9 @@
 REVERSE_PROXY_PORT=8000
 
-CFG_VERSION=latest
-CGW_VERSION=latest
-TXS_VERSION=latest
-UI_VERSION=latest
-EVENTS_VERSION=latest
+CFG_VERSION=v2.82.0
```
Comment (gabmontes) [inline .env.sample]:
```suggestion
CFG_VERSION=staging
```

File: container_env_files/cfg.env [hunk 2]
```diff
@@ -27,19 +27,18 @@ DOCKER_WEB_VOLUME=.:/app
...
 CGW_URL=http://nginx:8000/cgw
-CGW_FLUSH_TOKEN=some_super_random_token
+CGW_FLUSH_TOKEN=${CGW_FLUSH_TOKEN}
 
-CSRF_TRUSTED_ORIGINS="http://localhost:8000"
+CSRF_TRUSTED_ORIGINS="https://safe.hemi.xyz"
 
-# MEDIA_URL = "http://localhost:8000/cfg/media/"
-MEDIA_BASE_URL = "http://localhost:8000/cfg/media/"
+MEDIA_URL = "https://safe.hemi.xyz/cfg/media/"
```
Comment (gabmontes) [inline container_env_files/cfg.env]:
```suggestion
# MEDIA_URL = "http://localhost:8000/cfg/media/"

MEDIA_BASE_URL = "https://safe.hemi.xyz/cfg/media/"
```

File: container_env_files/cgw.env
Comment (gabmontes) [inline container_env_files/cgw.env:1]: Let's revert all the changes to this file except for the line defining `AUTH_TOKEN`.

**Review by gabmontes — CHANGES_REQUESTED** [review]

File: container_env_files/cgw.env
Comment (dhidalgX) [reply container_env_files/cgw.env:1]: updated

**Review by gabmontes — APPROVED** [review]

**Pull request merged**